src/components/BackgroundLayout/index.js

```diff
@@ -1,6 +1,16 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
I find the suffix `Layout` in `BackgroundLayout` a bit confusing as it would refer to layout stylesheets. How about `PageBackground`?

Reply:
Updated this and also the file re-organization in [bff8360](https://github.com/rafayet-monon/surveyor/pull/12/commits/bff8360a588303bbdaf43b1b3b276abbce7f1a3d) 🚀